---
title: 'Loss of copy number variant in RGS10 and GRK5 in human patient to potentially cause Horner Syndrome through increased neuronal inflammation'
author: "Melissa Chen"
date: '2021-04-29'
output:
  html_document:
    toc: yes
    toc_float:
      collapsed: no
      smooth_scroll: yes
  pdf_document:
    toc: no
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE, eval = TRUE, fig.align = "center", warning = F, message = F,
tidy=TRUE, tidy.opts=list(width.cutoff=60), R.options=list(max.print=100))
```

<P style="page-break-before: always">
\newpage

## Summary 

Horner Syndrome (HS) is a subtle, non-lethal disorder that is caused by lesions such as tumors or  inflammation to the oculosympathetic pathway (Martin, 2018). Symptoms of HS include droopy eyelids, constriction of the pupil, and displacement of the eye in the eye socket (Walton& Buono, 2003). There have been no genes identified that cause this disorder nor is the mode of inheritance well known. However, this paper seeks to identify the potential genetic causation of Horner Syndrome through analysis of DECIPHER patients. DECIPHER is a large database containing whole exome sequencing of patients with various phenotypes and mutations. Through analyzing DECIPHER patients and looking at de novo CNVs and genes with high probability of Loss-of-Function Intolerance (pLI) values, I propose that a deletion of *RGS10* and *GRK5* may cause HS. *RGS10* and *GRK5* both are more intolerant to loss of function variants and further research has shown that both genes inhibit excessive inflammation in neurons (Alhers-Dannen et al., 2020; Patial et al., 2011). A deletion of *RGS10* and *GRK5* in patient 406337 may cause increased inflammation and lead to lesions in the oculosympathetic pathway, ultimately causing HS.

## Introduction

Horner Syndrome (HS) is caused by a disruption of the oculosympathetic pathway to the eye and patients with HS have droopy upper eyelids (ptosis), constriction of the pupil (miosis), and displacement of the eye in the eye socket (enophthalmos) (Walton & Buono, 2003). This pathway starts in the hypothalamus and is composed of three neurons and traverses down the spinal cord, through the thorax, and to the eye socket (Martin, 2018). Lesions such as inflammation, trauma, and tumors can prevent normal neuron signaling, ultimately causing HS (Kynazer et al., 2017). The effects of HS are subtle and non-threatening, however the disruption in the neural signaling can lead to other diseases and phenotypes. While exact genetic basis and genes causing HS is unknown, researchers hypothesize HS can be inherited through family or idiopathically (Martin, 2018). 

Patient 406337 in DECIPHER is a male diagnosed with HS at age 14 and has a de novo loss of copy number variant (CNV) in their genome on chromosome 10 from bases 117884958-120232724, a 2.3 megabase (Mb) deletion. This loss of CNV resulted in the deletion of 33 genes and while there are other patients with a similar loss of CNV, they did not have HS. Therefore, it can be concluded that HS has a low penetrance. OMIM and the probability of loss-of-function (pLI) was used to narrow down the genes of interest. The higher the pLI value, the more likely the gene is haploinsufficient. Out of the 33 genes, there are a few with high pLI values so it would be a good starting place to find the genetic causation of HS. After research in Pubmed and OMIM, I predict genes *RGS10* and *GRK5* to cause HS. 

*RGS10* and *GRK5* are both seen to regulate inflammation through NF-κB signaling (Lee et al., 2013; Y. Zhang et al., 2015). In knockout mice, a homozygous deletion of *RGS10* resulted in an increase of inflammatory cytokines (TNF) and neuronal apoptosis suggesting that *RGS10* is a negative regulator of NF-κB signaling (Lee et al., 2013; Alqinyah et al., 2018). A loss of *RGS10* was seen to contribute to Parkinson’s disease, indicating that *RGS10* plays an important role in neuron signaling (Alhers-Dannen et al., 2020). On the other hand, *GRK5* is a positive regulator of the NF-κB signaling. In knockdown *GRK5* mice, there was reduced cytokine levels (Patiel et al., 2011; Packiriswamy et al., 2013). The role of both *RGS10* and *GRK5* in regulating neuronal inflammation suggests that a deletion of both genes will disrupt levels of inflammation, ultimately leading to HS. 

## Description of findings

Patient 406337 has a 2.3 Mb heterozygous de novo deletion of their DNA on chromosome 10 from bases 117884958-120232724, resulting in the deletion of 33 genes and a loss of function (LOF) mutation (Figure 1).  This patient has been diagnosed with 9 phenotypes: supernumerary nipples (additional nipples), hypospadias (urethra is on the underside of the penis), unilateral cryptorchidism (one testes does not descend normally into scrotum), misalignment of teeth, cystinuria (buildup of cystine in kidney and bladder), attention deficient hyperactivity disorder (ADHD), Horner Syndrome, ventriculomegaly (ventricles of brain are larger than normal), and a high, narrow palate. 

Comparisons between patient 406337 and other patients with the same deletion on chromosome 10 shows that there is little overlap in phenotype. Out of the 104 loss of CNV patients, only 10 of them have at least one matching phenotype, but none of them had HS (Figure 2). The lack of HS in other patients indicates that this phenotype has a low penetrance and may be caused by other genetic modifiers. 

![](/project/project4_files/Picture1.png)
Figure 1 *This screenshot shows the Genome Browser for patient 406337 from DECIPHER. The red bar in copy number variants indicates that this is a loss of CNV in chromosome 10.*

Genome Browser also showed data on CNVs within this genomic region but had limited data on how a loss of CNV would affect phenotypic expression (Figure 2). Most of the CNVs from Genome Browser were gain of copy number and the sole loss of copy number variant was diagnosed with global developmental delay like other patients with CNVs in DECIPHER. Therefore, even by looking at CNV in Genome Browser does not narrow down the genetic causation of HS. Additionally, Genome Browser offers information about the inheritance of the CNV, but the inheritance of the loss of CNV in red is unknown. However, information on gene expression for *RGS10* and *GRK5* can provide additional information about potential genetic causes. 

![](/project/project4_files/Picture2.png)
Figure 2 *This is a screenshot showing the CNVs of RGS10 and GRK5 and its phenotypic effects. There is only one documented deletion (loss of CNV) and that patient was diagnosed with global developmental delay.*

Since there is little commonality in phenotype between patient 406337 and other loss of CNV patients in both DECIPHER and Genome Browser, the pLI (probability of loss of intolerance) score for the 33 affected genes may provide insight to determine the genetic causation of HS. Out of the 33 deleted genes (Table 1), only 16 had any information on OMIM or pLI values. I narrowed down genes with high pLI and then performed literature search. The pLI value is a prediction of how tolerant a gene is to being haploinsufficient, a heterozygous loss of function (LOF) mutation. This value is calculated by seeing how frequent a protein truncating variation is seen from a sample of 125748 patients for each gene. The closer the value is to 1, the higher the probability of a gene being intolerant to a LOF mutation and the less frequent a gene is heterozygous. The genes more intolerant to a loss of function mutation are in Table 1. OMIM did not identify any of the genes in the previous sentence associated with similar phenotypes to HS. However, literature search from PubMed has research showing the role of *RGS10* and *GRK5* in regulating inflammation of neurons through immune responses. Additionally, graphs on *RGS10* and *GRK5* gene expression levels in different cell types gives an idea of where these genes play the biggest role (Figure 3). RGS10 is seen to have the highest transcripts per million (TPM) in Epstein-Barr Virus (EBV) transformed lymphocytes with 185.10 TPM. *GRK5* has the highest expression in artery cells, specifically in the aorta, with a TPM of 145.65. However, there is still a moderate amount of *GRK5* gene expression in EBV transformed lymphoma cells.

![](/project/project4_files/Picture3.png)
![](/project/project4_files/Picture4.png)
Figure 3 *A) This graph shows the levels of RGS10 expression in differentiated cells. The highest level of RGS10 expression is in EBV transformed lymphocytes with a median of 185.10 transcripts per million. B) This graph shows the levels of GRK5 expression in differentiated cells. The highest level of expression is in artery – aorta cells with 145.65 transcripts per million.*

## Discussion and Conclusion

Horner Syndrome is characterized by droopy eyelids, constricted pupils, and displacement of the eye within the eye socket (Martin, 2018). While there has not been a definitive gene identified to cause HS, most patients inherit the gene idiopathically although a few have inherited it from their parents. DECIPHER patient 406337 is diagnosed with HS and has a heterozygous loss of CNV in chromosome 10 bases 117884958-120232724 (2.3 Mb). Out of the 104 patients also diagnosed with a loss of CNV in this region, only 10 of them had at least one matching phenotype with patient 406337, but none of the shared phenotypes was HS. The lack of HS in other patients indicate that HS may have low penetrance and that there may be other genetic modifiers. Since DECIPHER is a database containing whole exome sequencing information of each patient, it may be that there is a mutation or loss of an important regulatory element of a gene causing HS, but that information is not shown. Whole genome sequencing would, however, show the sequence introns and regulatory elements. Therefore, it would provide additional information to potentially identify the causation of HS. 

The heterozygous loss of CNV in this region is quite large, spanning 2.3 Mb and deleting 33 genes (Table 1). It would be difficult to research all the genes, therefore, using the pLI value will help narrow down which genes to focus on. pLI measures the predicted tolerance of a gene to heterozygous loss-of-function mutation, or in other words, haploinsufficiency. This value does not directly identify genes that cause HS but narrows down on the genes that may cause HS. There are 2 genes, *RGS10* and *GRK5*, that are of interest to study. Literature on OMIM did not provide any phenotypes associated with any of the listed genes, however, PubMed has articles citing the expression of *RGS10* and *GRK5* in autoimmune cells such as microglia and lymphocytes. 

*RGS10* is primarily expressed in microglia, immune cells that act like macrophages, and negatively regulate inflammation through inhibiting nuclear factor κB (NF-κB) signaling (Lee & Tansey, 2016; Alqinyah et al., 2018). *RGS10* knockout mice exhibited increased pro-inflammatory cytokines due to increased expression of NF-κB target genes (Lee et al., 2013). In addition to microglia, *RGS10* is expressed in dopaminergic (DA) neurons, which are extremely sensitive to inflammation (Alhers-Dannen et al., 2020). Too much inflammatory stress on DA neurons can cause their degeneration. *GRK5* is another gene that is seen to be expressed in microglia and positively regulates anti-inflammatory cytokines through NF-κB signaling (Patial et al., 2011; Packiriswamy et al., 2013). A mice knockout of *GKR5* showed reduced levels of anti-inflammatory cytokines, suggesting that there will be increased inflammation. The increase in inflammation in *RGS10* and *GRK5* knockdown mice suggest that the regulation of autoimmune cells is important to maintain neuron health and prevent neuronal degeneration. 

Concordant with the literature, there is increased *RGS10* and *GRK5* gene expression in EBV transformed lymphocytes suggesting that there is an increased immune response towards EBV infection (Figure 3). EBV infected patients have been seen to have increased inflammation due to heightened T cell (lymphocyte) response (Johnson et al., 2019). Additionally, EBV patients saw a rise in NF-κB signaling (Takada et al., 2017), supporting the idea that increased inflammation results from dysregulation of autoimmune response (microglia and T cell activation) that potentially causes degradation of neurons. 

HS is caused by lesions to the oculosympathetic pathway, a three-neuron pathway starting at the hippocampus and ending at the eye socket (Martin, 2018; Walton & Buono, 2003). One of the mechanisms to cause HS is through inflammation (Knyazer et al., 2017). Since *RGS10* and *GRK5* are seen to play a role in regulating inflammation through NF-κB signaling in microglia, a deletion and knockdown of this region can lead to increased inflammation due to reduced anti-inflammatory cytokines and increased pro-inflammatory cytokines. Additionally, information on *RGS10* and *GRK5* expression in EBV transformed lymphocytes show that these genes potentially regulate autoimmune responses. EBV is shown to increase inflammation through recruitment of T cells and dysregulation of autoimmune responses from microglia or lymphocytes may lead to neuron degeneration. Therefore, I hypothesize that HS may be caused by deletion of *RGS10* and *GRK5* that lead to dysregulation of autoimmune cells, causing increased inflammation. *RGS10* and *GRK5* do not directly affect neuronal signaling or expression; it is the incapability of autoimmune cells to regulate and prevent excessive inflammation that may cause HS.  

The dysregulation of immune response leading to inflammation may explain the low penetrance of this syndrome because inflammation takes time to build up (chronic inflammation) and may not be seen in all patients. However, another reason for the low penetrance is that there may be other genetic modifiers causing HS. Nonetheless, *RGS10* and *GRK5* are interesting genes that affect the nervous system through regulation of the immune system and are good candidate genes to study. One point for future research to determine the causation of HS would be to gather sequencing information from the parents to determine its genetic cause. Some phenotypes occur more frequently from inheritance or de novo mutations. If one parent had the same genotype and HS as patient 406337, then it suggests that HS may arise due to a dominant mutation in a certain gene. Therefore, having triad information would allow better understanding of the genetic causes behind HS. 

*This project was completed in Spring 2021 for Human Genetics (BIO325T) with help from Dr. Steven Vokes.*

## References
 
Ahlers-Dannen, K. E., Spicer, M. M., & Fisher, R. A. (2020). Rgs proteins as critical regulators of motor function and their implications in parkinson’s disease. Molecular Pharmacology, 98(6), 730–738. https://doi.org/10.1124/mol.119.118836

Alqinyah, M., Almutairi, F., Wendimu, M. Y., & Hooks, S. B. (2018). Rgs10 regulates the expression of cyclooxygenase-2 and tumor necrosis factor alpha through a g protein-independent mechanism. Molecular Pharmacology, 94(4), 1103–1113. https://doi.org/10.1124/mol.118.111674

Johnson, D. B., McDonnell, W. J., Gonzalez-Ericsson, P. I., Al-Rohil, R. N., Mobley, B. C., Salem, J.-E., Wang, D. Y., Sanchez, V., Wang, Y., Chastain, C. A., Barker, K., Liang, Y., Warren, S., Beechem, J. M., Menzies, A. M., Tio, M., Long, G. V., Cohen, J. V., Guidon, A. C., … Balko, J. M. (2019). A case report of clonal EBV-like memory CD4+ T cell activation in fatal checkpoint inhibitor-induced encephalitis. Nature Medicine, 25(8), 1243–1250. https://doi.org/10.1038/s41591-019-0523-2 

Knyazer, B., Smolar, J., Lazar, I., Rosenberg, E., Tsumi, E., Lifshitz, T., & Levy, J. (2017). Iatrogenic horner syndrome: Etiology, diagnosis and outcomes. The Israel Medical Association Journal: IMAJ, 19(1), 34–38.

Lee, J.-K., Chung, J., Kannarkat, G. T., & Tansey, M. G. (2013). Critical role of regulator G-protein signaling 10 (Rgs10) in modulating macrophage M1/M2 activation. PloS One, 8(11), e81785. https://doi.org/10.1371/journal.pone.0081785 

Lee, J.-K., & Tansey, M. G. (2015). Physiology of rgs10 in neurons and immune cells. Progress in Molecular Biology and Translational Science, 133, 153–167. https://doi.org/10.1016/bs.pmbts.2015.01.005

Liu, J., Rasul, I., Sun, Y., Wu, G., Li, L., Premont, R. T., & Suo, W. Z. (2009). Grk5 deficiency leads to reduced hippocampal acetylcholine level via impaired presynaptic m2/m4 autoreceptor desensitization. Journal of Biological Chemistry, 284(29), 19564–19571. https://doi.org/10.1074/jbc.M109.005959

Martin, T. J. (2018). Horner syndrome: A clinical review. ACS Chemical Neuroscience, 9(2), 177–186. https://doi.org/10.1021/acschemneuro.7b00405

Packiriswamy, N., Lee, T., Raghavendra, P. B., Durairaj, H., Wang, H., & Parameswaran, N. (2013). G-protein-coupled receptor kinase-5 mediates inflammation but does not regulate cellular infiltration or bacterial load in a polymicrobial sepsis model in mice. Journal of Innate Immunity, 5(4), 401–413. https://doi.org/10.1159/000347002

Patial, S., Shahi, S., Saini, Y., Lee, T., Packiriswamy, N., Appledorn, D. M., LaPres, J. J., Amalfitano, A., & Parameswaran, N. (2011). G-protein coupled receptor kinase 5 mediates lipopolysaccharide-induced NFκB activation in primary macrophages and modulates inflammation in vivo in mice. Journal of Cellular Physiology, 226(5), 1323–1333. https://doi.org/10.1002/jcp.22460

Takada, H., Imadome, K.-I., Shibayama, H., Yoshimori, M., Wang, L., Saitoh, Y., Uota, S., Yamaoka, S., Koyama, T., Shimizu, N., Yamamoto, K., Fujiwara, S., Miura, O., & Arai, A. (2017). EBV induces persistent NF-κB activation and contributes to survival of EBV-positive neoplastic T- or NK-cells. PloS One, 12(3), e0174136. https://doi.org/10.1371/journal.pone.0174136 

Walton, K. A., & Buono, L. M. (2003). Horner syndrome: Current Opinion in Ophthalmology, 14(6), 357–363. https://doi.org/10.1097/00055735-200312000-00007

Zhang, Y., Shen, G., Shangguan, L., Yu, Y., & He, M. (2015). Involvement of NFκB signaling in mediating the effects of GRK5 on neural stem cells. Brain Research, 1608, 31–39. https://doi.org/10.1016/j.brainres.2015.02.041 

## Supplemental Data

![Table 1 *This is a compiled list of all 33 genes that were deleted from one copy of chromosome 10 of patient 406337. Only 16 out of the 33 genes had a pLI value.*]

```{r}
library(readr)
pli <- read_csv("C:/Users/shuen/OneDrive/pli.csv")
data.frame(pli)
```












