<!DOCTYPE html>
<html lang="en-US">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="author" content="Melissa Chen" />
    
    <link rel="shortcut icon" type="image/x-icon" href="/img/favicon.ico">
    <title>Epigenetics of FMR1 on Fragile X Syndrome</title>
    <meta name="generator" content="Hugo 0.79.0" />
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="/css/main.css" />
    <link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" />
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,400,200bold,400old" />
    
    <!--[if lt IE 9]>
			<script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
			<script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
		<![endif]-->

    
  </head>

  <body>
    <div id="wrap">
      
      <nav class="navbar navbar-default">
  <div class="container">
    <div class="navbar-header">
      <a class="navbar-brand" href="/"><i class="fa fa-home"></i></a>
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
    </div>
    <div class="navbar-collapse collapse" id="navbar">
      <ul class="nav navbar-nav navbar-right">
      
        
        <li><a href="/post/">BLOG</a></li>
        
        <li><a href="/projects/">PROJECTS</a></li>
        
        <li><a href="/resume/">RESUME</a></li>
        
      
      </ul>
    </div>
  </div>
</nav>

      <div class="container">
        <div class="blog-post">
          <h3>
            <strong><a href="/project/project5/">Epigenetics of FMR1 on Fragile X Syndrome</a></strong>
          </h3>
        </div>
 
<div class="blog-title">
          <h4>
         April 28, 2021 
            &nbsp;&nbsp;
            
          </h4>
        </div>

        <div class="panel panel-default">
          <div class="panel-body">
            <div class="blogpost">
              
<link href="/rmarkdown-libs/anchor-sections/anchor-sections.css" rel="stylesheet" />
<script src="/rmarkdown-libs/anchor-sections/anchor-sections.js"></script>


<P style="page-break-before: always">
<div style="page-break-after: always;"></div>
<p>Epigenetics is the study of heritable changes in phenotype without changing DNA nucleotide sequence (Gibney &amp; Nolan, 2010). It has become a popular field of research where scientists try to better understand the impact of epigenetics and gene expression on human diseases. Epigenetic modifications can be found on both the DNA and histones. Histones are positively charged proteins that assist in chromatin folding, which can actively express or repress gene expression. Similarly, DNA acetylation and methylation recruit activator and repressor complexes that regulate transcription. <em>FMR1</em> is a gene on the X chromosome that is known to cause the inheritable disease Fragile X Syndrome (FXS) due to epigenetic modifications (Kraan et al., 2019). FXS is characterized by intellectual disability caused by an expanded 5’UTR CGG region and increased 5mC methylation. The increased amount of DNA and histone methylation, specifically H3K9me3, reduces the production of functional fragile X retardation protein (FMRP) through condensing the chromatin (Brasa et al., 2016). Differences in regions of methylation and histone modifications cause varying levels of symptom severity and provide scientists with new ideas for potential treatment. Identification of novel methylation sites (5mC) and histone modifications (H3K9me3) of <em>FMR1</em> can provide a better understanding of the mechanisms behind FXS and introduce treatment plans targeting the removal of methylation and CGG repeats through drugs and CRISPR/Cas9 respectively.</p>
<p>Fragile X Syndrome is an inherited disease characterized by intellectual disability caused by an expanded 5’UTR CGG region (Kraan et al., 2019). There are two classes of defective <em>FMR1</em> alleles, premutation alleles with &lt;200 CGG repeats and full mutation alleles with &gt;200 CGG repeats. Premutation alleles only confer an increased risk of <em>FMR1</em> related disorders while full mutation alleles directly show neurodegenerative effects in patients. Neurodegenerative effects include intellectual disability, speech impairment, and anxiety. In addition to those symptoms, patients are also often diagnosed with autism. The expanded repeat regions induce changes in the epigenetic marks near the promoter region of the <em>FMR1</em> gene, which results in repressed transcriptional expression and reduction of functional fragile X retardation protein (FMRP). In addition to the CGG repeats, the methylation of histones and CpG islands also contribute to the reduced transcription of functional FMRP (Brasa et al., 2016).</p>
<p>As mentioned in the previous paragraph, patients with FXS have extended CGG repeat regions in the promoter region of <em>FMR1</em> and these repeats are thought to cause FXS through de novo methylation and DNA/RNA structures (Hecht et al., 2017; Kraan et al., 2019). De novo methylation occurs during embryogenesis, so researchers studied the mechanisms behind methylation in embryonic mice cells and yeast (Hecht et al., 2017). They hypothesized that the defective <em>FMR1</em> gene is inactivated through Dicer and AGO proteins and not directly through de novo methylation of CGG repeats. Looking at previous research in yeast, Dicer forms siRNA by cleaving double-stranded RNA (dsRNA), which is then recognized by AGO proteins. AGO proteins recognize siRNA and upon interaction, the AGO-siRNA complex will transcriptionally repress <em>FMR1</em> by binding complementary to the CGG repeats. ChIP analysis was performed to see if there is a significant level of AGO1 in mice ES cells and if they were bound to CGG repeat siRNA. The analysis confirmed that AGO1 is indeed bound to fragments of CGG repeat siRNA through immunoprecipitation. However, in normal mice ES cells, AGO1 was not found to be bound to CGG repeat siRNA. The difference in AGO1 presence and binding in normal versus mutant <em>FMR1</em> cells shows that it plays a function in the repression of <em>FMR1</em> expression specifically using the CGG siRNA as a guide to complementary bind to the repeats and inhibit transcription.</p>
<p>As mentioned before, Dicer forms siRNA through cleaving dsRNA, and researchers wanted to confirm the function of siRNA using mice as a model organism. However, in mice ES cells, there is no sign of dsRNA so there must be some other structure that functions similarly to dsRNA. As it turns out, long CGG repeats can form a hairpin structure that has a high affinity to Dicer, so Dicer will instead process the hairpin structure to form siRNA (Hecht et al., 2017). Therefore, it is highly likely that CGG repeats in human cells will also form hairpin structures to be cut by Dicer. Northern blot analysis and RNA-seq was performed on FXS embryonic stem cells (ES) and confirmed CGG repeat siRNAs that are 19-23 nt long. In comparison, the normal human ES cells did not show a distinct band of 21 nt nor did the RNA-seq identify significant levels of CGG repeat siRNA. The difference between the presence of CGG repeat siRNA and AGO1 in FSX ES cells and normal ES cells strongly supports the hypothesis that the Dicer/AGO system transcriptionally represses <em>FMR1</em>. Furthermore, the experimental results explained a potential mechanism in how CGG repeats cause FXS and introduced new ways to treat FXS by reducing the number of CGG repeats or by inhibiting Dicer or AGO.</p>
<p>In addition to CGG repeats, another way the <em>FMR1</em> gene can be epigenetically modified is through histone modification. Histone methylation transcriptionally represses <em>FMR1</em> by condensing the chromatin and preventing transcriptional factors from binding to <em>FMR1</em>. A clinical study used ChIP-seq and qPCR to see changes in histone modification markers in FSX patients. There were active histone modifications (H3K4me2, H3K9ac, H4K16ac, H4K20me1) and increased repressive histone modifications (H3K9me3, H4K20me3) around the transcriptional start site (Brasa et al., 2016). H3K9me3 and H3K29me1 are associated with heterochromatinization and prevent transcription of <em>FMR1</em> since transcription factors would not be able to bind. H3K9me3 modifications were also mentioned when discussing the Dicer/AGO1 system (Hecht et al., 2017). Consistent with the clinical study, ChIP seq saw that there was increased enrichment of H3K9me3 and heterochromatin protein B (HPB) in the 5’ region of the Fragile X <em>FMR1</em> gene. The protein that assists in heterochromatin formation is called SUV39H and helps maintain the H3K9me3 enrichment with the help of DNMT3a, which is a writer protein that methylates a DNA region for the first time. In the early stages of development, <em>FMR1</em> is not methylated but becomes so after cell differentiation and inactivation of the X chromosome through SUV39H and DNMT3a. SUV39H null cells lacked H3K9me3 enrichment and have decreased DNA methylation. Therefore, in addition to the Dicer/AGO interactions that directly repress transcription of <em>FMR1</em>, SUV39H is recruited and indirectly methylates and represses <em>FMR1</em> by tri methylating H3K9 and recruiting DNMT3a to methylate the DNA. The interaction of SUV39H and DNMT3a gives an additional pathway to repress <em>FMR1</em> transcription through heterochromatinization. Additionally, identification of regions with increased H3K9me3 modifications and DNA methylation can provide useful in identifying FXS in addition to recognition of the CGG repeats.</p>
<p>Finally, <em>FMR1</em> can also be epigenetically modified through methylation of the CpG islands with 5mC (Brasa et al., 2016). DNA writers methylate the CpG islands of the DNA through de novo methylation or maintenance (Kumar et al., 2018). An example of a de novo writer is DNMT3a which will methylate CpG islands for the first time. However, epigenetic marks can also be passed down from one generation of cells to the next through maintenance writers. Maintenance writers (DNMT1) will recognize hemimethylated strands of DNA and use the methylated strand as a guide to methylate newly synthesized DNA strand. In the clinical study mentioned in the previous paragraph, researchers also wanted to identify novel regions of 5mC modification that can be used in the identification of FXS (Brasa et al., 2016). Methylated DNA immunoprecipitation was used to profile and identify regions of 5mC binding and resulted in finding a correlation between the amount of 5mC binding and the severity of FXS. Most of the 5mC binding occurs in the active <em>FMR1</em> gene body, regions surrounding the TSS, and active enhancers. 5mC binding in these regions ranges anywhere from 50-80% of the enriched region. The increased methylation at these regions of the DNA is not surprising since all are involved in the active transcription of FMRP, and methylation will prevent transcription and expression. Between the eight patients studied, all of them had consistent 5mC modification levels around the TSS of <em>FMR1</em> and identified another methylation marker 5hmC that was correlated inversely with both 5mC and FXS severity. The measurement of 5mC and 5hmC levels may be another way to characterize and diagnose FXS.</p>
<p>Looking at both histone methylation patterns and 5mC binding, repression of <em>FMR1</em> is associated with increased 5mC and H3K9me3 while activation is associated with 5hmC and activator histone marks (Brasa et al., 2016). This introduces a potential new interaction between DNA methylation and histone post-translational modifications (PTM). In the Dicer/Ago system, SUV39H was identified as an enzyme that helps tri methylate H3K9 and recruits DNMT3a (Hecht et al., 2017). However, there was not much known about the specifics of how methylation of DNA by DNMT3a can regulate FXS and so more research can be done in this direction to identify the role of DNMT3a and 5mC. Overall, 5mC enrichment can be a novel way to characterize and predict FXS severity in addition to being a target for treatment.</p>
<p>Understanding the epigenetic modifications causing FXS allows for innovative treatments. Current treatments for FXS include symptomatic, targeted, and etiological approaches (Shitik et al., 2020). Targeted approaches for FXS include using compounds to inhibit the function of certain proteins or complexes that cause FXS. Symptomatic approaches are similar in that drugs are used to treat FXS, but in this case, it is used to treat the symptoms of FXS and not the underlying cause. An etiological approach removes epigenetics markings such as DNA and histone methylation and can directly allow transcription again (Kraan et al., 2019; Shitik et al., 2020). Some developed drugs that remove DNA methylation include 5-azadeoxycytidine (5-azaC) and can partially restore FMRP production. However, a caveat of using 5-azaC is that it has toxic effects and patients will experience side effects such as nausea and diarrhea. Another potential but developing treatment would use non-coding RNA. Noncoding RNA (ncRNA) is involved in heterochromatinization and DNA methylation so using noncoding RNA to complementarily bind to ncRNA will prevent heterochromatinization. However, there has not been clinical testing done to test if ncRNA can inhibit CGG siRNA or other RNA strands. So, while this mode of treatment may help treat FXS, no data has been shown to prove its efficiency. Finally, another approach is to use gene therapy to make changes directly in the genome of the patient by removing excess CGG repeats.</p>
<p>The first possible treatment would be to use 5-aza. 5-aza has been discovered to remove 5mC methylation on CpG islands through binding to DNMT1 and preventing it from binding onto the DNA (Shitik et al., 2020). Interestingly enough, a study has been done to see the specificity of 5-azaC in demethylating regions of DNA and through bisulfite sequencing, it is seen that 5-azaC unmethylated the promoter region, while the upstream region was untouched (Tabolacci et al., 2016). Additionally, <em>FMR1</em> was 30-40% reactivated for about a month after 7-day treatment of 5-azaC, and levels of transcription peaked days after the last treatment. Since the wild-type cell line did not exhibit significant changes in <em>FMR1</em> expression, it can be inferred that 5-azaC may be used in moderate amounts for treatment. However, despite knowing that 5-azaC does not randomly unmethylated regions of DNA and can temporarily induce <em>FMR1</em> expression, it is not the best method for treatment due to the multiple side effects of taking this drug.</p>
<p>Another valid method to treat genetic diseases is through CRISPR/Cas9, a system that can remove pieces of DNA (Park et al., 2015). CGG repeats have been removed successfully from human induced pluripotent stem cells (iPSC) using Cas9 and single-guide RNA (sgRNA). sgRNA guides the CRISPR/Cas9 complex to targeted complementary DNA regions and in this case, was a strand of RNA that codes for the CGG repeats in FXS patients. In addition to the removal of CGG repeats, researchers also sequenced the DNA for changes in epigenetic markings and imaged the cell to see changes in chromatin structure. <em>FMR1</em> was observed to be transcriptionally active after the removal of CGG repeats and the promoter region was seen to have less methylation. Therefore, CRISPR/Cas9 may be a useful treatment to remove CGG repeats and reactive <em>FMR1</em> since human IPSCs were seen to have a more relaxed chromatin structure, reduced methylation, and increased <em>FMR1</em> transcription. However, one limitation of using CRISPR/Cas9 is the risk of off-target mutations which are mutations that result from CRISPR/Cas9 cleaving highly homologous regions of DNA and leads to additional mutations (Zhang et al., 2014). These mutations can lead to errors in transcription and cell death so it is important to increase the specificity of the sgRNA to limit off-target mutations. Additionally, there has to be sufficient sgRNA produced by the host cell for CRISPR/Cas9 to function. However, it is often difficult for the host to produce enough sgRNA with RNA Pol II in addition to the difficult insertion of the complex. These two issues are fundamental to ensure CRISPR/Cas9 functions efficiently, so additional research should be done to see how viable it would be to use CRISPR/Cas9 on a large scale in vivo.</p>
<p>The three epigenetic pathways characterizing FXS are the CGG repeats, H3K9me3 modifications, and 5mC enrichment around the TSS of <em>FMR1</em>. CGG repeats inhibit transcription by utilizing the Dicer/AGO system to create CGG siRNA that bind complementarily to the repeats. Additionally, SUV39H and DNMT3a also are recruited and indirectly inactivate <em>FMR1</em> by causing heterochromatinization and methylating the DNA. By identifying and characterizing FXS with novel regions of 5mC enrichment and histone tail modifications gives researchers another direction in research to find the specifics of how each causes FXS. Altogether, these epigenetic pathways and markings are potential targets for treatment. Potential methods of treatment include using drugs such as 5-azaC to demethylate 5mC and allow the TSS and active enhancer regions to be open for transcription. The other method to treat FXS is through cutting out additional CGG repeats by CRISPR/Cas9. Although all of these treatments target a specific modification or mechanism that results in FXS, they have to be more refined in application to not cause global disruption in epigenetic markings. There leaves much to be discovered about FXS, the mechanisms behind its inhibition and mutation, and potential treatments, but using the knowledge of this epigenetic disease, researchers can apply certain concepts to other diseases.</p>
<p><em>This project was completed in Spring 2021 for Genetics Lab (BIO325L) with help from Dr. Katie Bruner and Amanda Vines.</em></p>
<div id="references" class="section level2">
<h2>References</h2>
<p>Brasa, S., Mueller, A., Jacquemont, S., Hahne, F., Rozenberg, I., Peters, T., He, Y., McCormack, C., Gasparini, F., Chibout, S.-D., Grenet, O., Moggs, J., Gomez-Mancilla, B., &amp; Terranova, R. (2016). Reciprocal changes in DNA methylation and hydroxymethylation and a broad repressive epigenetic switch characterize FMR1 transcriptional silencing in fragile X syndrome. Clinical Epigenetics, 8(1), 15. <a href="https://doi.org/10.1186/s13148-016-0181-x" class="uri">https://doi.org/10.1186/s13148-016-0181-x</a></p>
<p>Gibney, E. R., &amp; Nolan, C. M. (2010). Epigenetics and gene expression. Heredity, 105(1), 4–13. <a href="https://doi.org/10.1038/hdy.2010.54" class="uri">https://doi.org/10.1038/hdy.2010.54</a></p>
<p>Hecht, M., Tabib, A., Kahan, T., Orlanski, S., Gropp, M., Tabach, Y., Yanuka, O., Benvenisty, N., Keshet, I., &amp; Cedar, H. (2017). Epigenetic mechanism of FMR1 inactivation in Fragile X syndrome. The International Journal of Developmental Biology, 61(3-4–5), 285–292. <a href="https://doi.org/10.1387/ijdb.170022hc" class="uri">https://doi.org/10.1387/ijdb.170022hc</a></p>
<p>Kumar, S., Chinnusamy, V., &amp; Mohapatra, T. (2018). Epigenetics of modified dna bases: 5-methylcytosine and beyond. Frontiers in Genetics, 9, 640. <a href="https://doi.org/10.3389/fgene.2018.00640" class="uri">https://doi.org/10.3389/fgene.2018.00640</a></p>
<p>Kraan, C. M., Godler, D. E., &amp; Amor, D. J. (2019). Epigenetics of fragile X syndrome and fragile X-related disorders. Developmental Medicine &amp; Child Neurology, 61(2), 121–127. <a href="https://doi.org/10.1111/dmcn.13985" class="uri">https://doi.org/10.1111/dmcn.13985</a></p>
<p>Park, C.-Y., Halevy, T., Lee, D. R., Sung, J. J., Lee, J. S., Yanuka, O., Benvenisty, N., &amp; Kim, D.-W. (2015). Reversion of fmr1 methylation and silencing by editing the triplet repeats in fragile x ipsc-derived neurons. Cell Reports, 13(2), 234–241. <a href="https://doi.org/10.1016/j.celrep.2015.08.084" class="uri">https://doi.org/10.1016/j.celrep.2015.08.084</a></p>
<p>Shitik, E. M., Velmiskina, A. A., Dolskiy, A. A., &amp; Yudkin, D. V. (2020). Reactivation of FMR1 gene expression is a promising strategy for fragile X syndrome therapy. Gene Therapy, 27(6), 247–253. <a href="https://doi.org/10.1038/s41434-020-0141-0" class="uri">https://doi.org/10.1038/s41434-020-0141-0</a></p>
<p>Tabolacci, E., Mancano, G., Lanni, S., Palumbo, F., Goracci, M., Ferrè, F., Helmer-Citterich, M., &amp; Neri, G. (2016). Genome-wide methylation analysis demonstrates that 5-aza-2-deoxycytidine treatment does not cause random DNA demethylation in fragile X syndrome cells. Epigenetics &amp; Chromatin, 9(1), 12. <a href="https://doi.org/10.1186/s13072-016-0060-x" class="uri">https://doi.org/10.1186/s13072-016-0060-x</a></p>
<p>Zhang, F., Wen, Y., &amp; Guo, X. (2014). CRISPR/Cas9 for genome editing: Progress, implications and challenges. Human Molecular Genetics, 23(R1), R40–R46. <a href="https://doi.org/10.1093/hmg/ddu125" class="uri">https://doi.org/10.1093/hmg/ddu125</a></p>
</div>

            
        <hr>         <div class="related-posts">
                <h5>Related Posts</h5>
                
              </div> 
            </div>
          </div>

   <hr>  <div class="disqus">
  <div id="disqus_thread"></div>
  <script type="text/javascript">

    (function() {
      
      
      if (window.location.hostname == "localhost")
        return;

      var disqus_shortname = '';
      var dsq = document.createElement('script'); dsq.type = 'text/javascript'; dsq.async = true;
      dsq.src = '//' + disqus_shortname + '.disqus.com/embed.js';
      (document.getElementsByTagName('head')[0] || document.getElementsByTagName('body')[0]).appendChild(dsq);
    })();
  </script>
  <noscript>Please enable JavaScript to view the <a href="http://disqus.com/?ref_noscript">comments powered by Disqus.</a></noscript>
  <a href="http://disqus.com/" class="dsq-brlink">comments powered by <span class="logo-disqus">Disqus</span></a>
</div> 
        </div>
      </div>
    </div>

    
    <footer>
  <div id="footer">
    <div class="container">
      <p class="text-muted">&copy; All rights reserved. Powered by <a href="https://gohugo.io/">Hugo</a> and
      <a href="http://www.github.com/nurlansu/hugo-sustain/">sustain</a> with ♥</p>
    </div>
  </div>
</footer>
<div class="footer"></div>


<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>

<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa" crossorigin="anonymous"></script>
<script src="/js/docs.min.js"></script>
<script src="/js/main.js"></script>

<script src="/js/ie10-viewport-bug-workaround.js"></script>


    
  </body>
</html>
